test/Dunet.Test.csproj

```diff
@@ -22,7 +22,8 @@
     </ItemGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\src\Dunet.csproj" />
+        <ProjectReference Include="..\src\Dunet\Dunet.csproj" />
+        <ProjectReference Include="..\src\Dunet.Generator\Dunet.Generator.csproj" />
```

Comment:
Do we need this reference if `Dunet.csproj` references it?

Reply:
Yeah, for the test project we do, as Dunet.csproj doesn't forward the reference to Dunet.Generator.csproj, the same is true with the Benchmark project